### PR TITLE
Correct Antora structure

### PR DESCRIPTION
**Pull request by mendonk**
* Restructure pulsar-sink-docs repo to be in parity with other DS docs content. 
* Builds locally with `antora --fetch --clean --stacktrace  playbooks/site-local-pulsarConnector.yaml` with no errors.
* Scripsi branch: https://github.com/riptano/scripsi/tree/pulsar-connector-update 
* Related to https://github.com/datastax/pulsar-sink-docs/pull/15

### CODE REVIEW

File: playbooks/site-publish-pulsarConnector.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  # set to "~/work/<repo-name>/build/site". This is because bsys clones the
+  # content source repo into "/work", and then looks for the Antora-generated
+  # site in "build/site" within the repo directory when deploying/syncing.
+  dir: '~/work/pulsar-sink-docs/build/site'
+
+content:
+  edit_url: false
+  sources:
+  - url: '~/work/pulsar-sink-docs'
+    branches: main
```
Comment (eric-schneider) [inline playbooks/site-publish-pulsarConnector.yaml:21]: NOTE: I tested this PR using plain vanilla bsys, and not the streaming site. So please correct me if my feedback doesn't apply.

I rebuilt bsys using your changes in https://github.com/riptano/scripsi/pull/30, and then ran the following command:

```
bsys -v build -o branch=antora-structure,deployDir=en/antora-structure pulsarConnector
```

Antora failed with the following error:

```
[21:40:12.320] FATAL (antora): the start path 'docs-src/pulsar-sink-core' does not exist in /home/writer/work/pulsar-sink-docs (ref: remotes/origin/main)
```

Although bsys initially checks out the correct branch (`antora-structure`), it then reads through the playbook and sees `branches: main` and switches back to `main` before the build starts. And since `main` doesn't have the correct changes in it, the build fails.

**tl;dr The content source in the playbook must have `branches: HEAD` in order for `bsys build -o` to work. (Note that there is an exception to this when building a "multi-repo" docset such as [astra-docs](https://github.com/riptano/astra-docs/blob/main/playbooks/site-publish-serverless.yaml#L20) -- not to be confused with a unified site like the streaming site.)**

```suggestion
    branches: HEAD
```

**Review by eric-schneider — CHANGES_REQUESTED** [review]
Found a potential issue. Please review my testing methodology.

File: playbooks/site-publish-pulsarConnector.yaml (hunk 1 above)
Comment (mendonk) [reply playbooks/site-publish-pulsarConnector.yaml]: @eric-schneider 
Builds to coppi with HEAD and kebab case 
https://coppi.aws.dsinternal.org/en/antora-structure/docs/index.html

**Review by eric-schneider — APPROVED** [review]
I pushed a commit with some changes: https://github.com/datastax/pulsar-sink-docs/pull/16/commits/aa65d87929ec4809d8a3864e417fb1bd0ac18442

I was then able to build successfully with the scripsi changes in https://github.com/riptano/scripsi/pull/31
